features/doctrine/exists_filter.feature

```diff
@@ -0,0 +1,59 @@
+Feature: Exists filter on collections
+  In order to retrieve large collections of resources
+  As a client software developer
+  I need to retrieve collections that exists
+
+  @createSchema
+  Scenario: Get collection where exists does not exist
+    Given there is "15" dummy objects with dummyBoolean true
+    When I send a "GET" request to "/dummies?dummyBoolean[exists]=0"
+    Then the response status code should be 200
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/ld+json; charset=utf-8"
+    And the JSON should be valid according to this schema:
+    """
+    {
+      "type": "object",
+      "properties": {
+        "@context": {"pattern": "^/contexts/Dummy$"},
+        "@id": {"pattern": "^/dummies$"},
+        "@type": {"pattern": "^hydra:Collection$"},
+        "hydra:totalItems": {"type":"number", "maximum": 0},
+        "hydra:member": {
+          "type": "array",
+          "maxItems": 0
+        },
+        "hydra:view": {
+          "@id": {"pattern": "^/dummies\\?dummyBoolean[exists]=0"},
```

Comment:
This json schema is not correctly constructed. You could put anything in the pattern, tests would still pass.
See #967
(Same goes bellow)